virtio-devices/src/epoll_helper.rs

```diff
@@ -25,6 +25,7 @@ pub enum EpollHelperError {
     CreateFd(std::io::Error),
     Ctl(std::io::Error),
     Wait(std::io::Error),
+    IoUringRegisterEventFd(std::io::Error),
```

Comment:
I don't think this should be here. This is about errors that the `EpollHelper` code can return. I see you return this in `run()` which is a bad idea as the errors in `run()` are not currently surfaced.

Reply:
Done!